.github/workflows/test.yaml

```diff
@@ -76,10 +76,6 @@ jobs:
           yarn install --frozen-lockfile
           yarn eslint --max-warnings=0
           yarn tslint
-          yarn prettier-fix
```

Comment:
**(non-blocking):** it would be nice to have a blocklist / allowlist for this fix path, so that the changes made from github ui can still work. I know that I'll need to create a new commit for the CI to run, but I'd prefer that over having to check out and run this locally for each branch.

Reply:
It can be done by having a new step, and protecting that with an `if` something like so:
```
if: github.actor != 'user1' && github.actor != 'user2'
```
or
```
if: contains(github.event.pull_request.labels.*.name, 'Autoformat')
```

Reply:
SGTM, will disable now to get CI flowing again, then tag me and I will approve this